app/controllers/payment_controller.rb

```diff
@@ -1,3 +1,4 @@
+# CR: [Michał] W railsach prawie nigdy nie robimy require
```

Comment:
Nie zgadzam się
To są elementy standardowej biblioteki, dodawanie ich do Gemfile byłoby bez sensu.
Nie powinno się robić `require` na pliki z własnym kodem, które są w ścieżkach auto-ładowania `zeitwerk`.

Ale biblioteki (tym bardziej ze stdlib) jak najbardziej można require'ować.